src/lua/widget/container.c

```diff
@@ -88,7 +88,7 @@ static int child_added(lua_State *L)
   lua_widget widget;
   luaA_to(L, lua_widget,&widget,2);
   lua_getuservalue(L,1);
-  luaA_push(L,lua_widget,&widget);
+  lua_pushlightuserdata(L, widget->widget);
```

Comment:
I'm pretty sure it's not functionnaly equivalent. Here we use raw `gtk_widget` instead of `lua_widget` as a key in uservalue. So as a result container uservalue contains mapping `gtk_widget -> lua_widget`. At the end we can easly look up for child lua_widget by gtk searched_widget, i.e. ` return container->user_value[searched_widget]`
 See the end of container_numindex function

```c
 static int container_numindex(lua_State*L)
 {
//...
   } else {
     GtkWidget *searched_widget = g_list_nth_data(children,index);
     g_list_free(children);
     lua_getuservalue(L,1);
     lua_pushlightuserdata(L,searched_widget);
     lua_gettable(L,-2);
     return 1;
   }
}
```  

So this is just a fix of the regression introduced by 0edd2f429b905e86ce5dbc6cbd3db1beea5d4c1f, I'm open to fix this other way if you have another proposition.